packages/zoltan2/src/algorithms/zoltan/Zoltan2_AlgZoltan.hpp

```diff
@@ -54,7 +54,8 @@
 #include <Zoltan2_Model.hpp>
 
 #include <Zoltan2_AlgZoltanCallbacks.hpp>
-#include <zoltan_cpp.h>
```

Comment:
Including zoltan_cpp.h should remain here.  This file uses the C++ capabilities of Zoltan.  rcb_partition_tree.c and rcb_partition_tree.h do not (or, at least, should not).